src/commands/documentation/docs.js

```diff
@@ -0,0 +1,43 @@
+import { Command } from 'discord.js-commando'
+
+import links from '../../../data/documentation'
+
+module.exports = class DocsDocsCommand extends Command {
```

Comment:
`DocsCommand`?

Reply:
<CommandGroup><CommandName>Command (I've asked this before)

Reply:
My own personal convention is:

```
NAME_OF_COMMAND_GROUP + NAME_OF_COMMAND + COMMAND.
```

- The command name is `docs`.
- The command group is docs (well by now it's actually `documentation`)
- So it is DocsDocsCommand (should be DocumentationDocsCommand by now).

I think this command should be renamed to Vue anyway and that there should be a command per library.

So we are left with:

- DocumentationVueCommand
- DocumentationVueXCommand
- DocumentationVueRouterCommand

Thoughts? @Elfayer